firedrake/mesh.py

```diff
@@ -34,6 +34,10 @@
 }
 
 
+unmarked = -1
+"""A mesh marker that selects all entities that are not explicitly marked."""
```

Comment:
When I write `dS(foo)` to select a subdomain, `foo` must be an integer.  So I use `-1` (the default for unmarked entities in DMLabels) to indicate the complement of all marked entities here.